lib/training-api.ts

```diff
@@ -25,8 +25,17 @@ export async function getUserInfo(username: string): Promise<UserInfo | null> {
       credentials: "include",
     });
     if (req.status !== 200) return null;
-    const data = await req.json();
+    var data = await req.json();
     if (data.success !== 1) return null;
+    const req_terr = await fetch("https://territoriali.olinfo.it/api/user/" + username + "/scores", {
+      method: "GET",
+    });
+    if (req_terr.status != 200) return null;
+    const terr_data = await req_terr.json();
+    for (var terr_task of terr_data) {
+      terr_task['score'] = terr_task['score'] * 100 / terr_task['max_score'];
```

Comment:
```suggestion
      terr_task['score'] = terr_task['score'] * 50 / terr_task['max_score'];
```

We decided to make terry tasks worth 50 points.

Reply:
Other parts of the UI assume you can get up to 100 points in each task.
I think we can fix this later, and have them worth 100 points for now - it is sufficient for getting started on training for the regional competition.